sw/device/lib/crypto/api.h

```diff
@@ -0,0 +1,1382 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_CRYPTO_API_H_
+#define OPENTITAN_SW_DEVICE_LIB_CRYPTO_API_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
```

Comment:
Can we add a comment indicating that this is a work in progress?

Reply:
Done.